lib/exercism/db_source.rb

```diff
@@ -0,0 +1,12 @@
+class DbSource
+	attr_reader :submission
+
+	def initialize(submission)
+		submission = submission
```

Comment:
Useless assignment to variable - `submission`.<br>Use 2 (not 1) spaces for indentation.<br>Tab detected.